src/frontend/komponenter/Behandling/Inngangsvilkår/Aleneomsorg/typer.ts

```diff
@@ -0,0 +1,104 @@
+export interface IBarnMedSamvær {
```

Comment:
Er det denne som skal brukes til flere inngangsvilkår eller burde den kanskje hete noe med `Aleneomsorg`?

Reply:
Denne skal brukes til flere vilkår, ja. Aleneomsorg er egentlig ikke særlig beskrivende for datainnholdet. Det er Barn, Annen forelder og samværsdetaljer her.